### ISSUE TITLE
StringVB -> C#: Single initialisation

### ISSUE THREAD

**Issue by marcal** [icsharpcode/CodeConverter#548]
Hello,

Here is a new error with the version 8.0.1.0
### Input code
```vbnet
dim TestSingle as single
TestSingle=29.7
```

### Erroneous output
```C#
single TestSingle;
TestSingle=29.7;

With an error (you must add F  to the sufix so it must be TestSingle=29.7F;


Marc

**Comment by davidkdb** [comment]
The same thing happens with other decimal types.

**Comment by marcal** [comment]
Yes (but not with double). Also if you put 21 there is no problem

**Comment by Saibamen** [comment]
Should be fixed in release 8.0.2: https://github.com/icsharpcode/CodeConverter/releases/tag/8.0.2
Please retest

**Comment by marcal** [comment]
Hello,

It is fixed. Thank you
